support/index.html

```diff
@@ -3,45 +3,163 @@
+      <div class="col-sm-12">
+        <div id="btn-clear-log" class="btn btn-danger">Clear logs</div>
+      </div>
+      <div class="col-sm-12" style="margin-top: 10px; max-height: 800px; overflow: auto;">
+        <pre>
+          <code id="logger" class="language-js"></code>
+        </pre>
+      </div>
+    </div>
+  </main>
+  <script src="https://unpkg.com/jquery@3.2.1/dist/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.6.0/prism.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.6.0/components/prism-javascript.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.6.0/plugins/normalize-whitespace/prism-normalize-whitespace.min.js"></script>
   <script src="/build/lock.js"></script>
+  <script id="helper-scripts">
+    //this is not related to Auth0. It's just here to make the page work correctly.
+    Prism.plugins.NormalizeWhitespace.setDefaults({
+      'remove-trailing': true,
+      'remove-indent': true,
+      'left-trim': true,
+      'right-trim': true,
+    });
+    var logs = [];
+    if (window.localStorage.auth0logs) {
+      logs = JSON.parse(window.localStorage.auth0logs);
+    }
+    function printLogs() {
+      window.localStorage.auth0logs = JSON.stringify(logs);
+      var code = '';
+      logs.forEach(function (l) {
+        code += '\r\n' + JSON.stringify(l, null, 1);
+      });
+      var html = Prism.highlight(code, Prism.languages.javascript);
+      $('#logger').html(html);
+    }
+    function clearLogs() {
+      logs = [];
+      window.localStorage.removeItem('auth0logs');
+      printLogs();
+    }
+    function subscribeToEvents(instance) {
+      var validEvents = [
+        'show',
+        'hide',
+        'unrecoverable_error',
+        'authenticated',
+        'authorization_error',
+        'hash_parsed',
+        'signin ready',
+        'signup ready',
+        'forgot_password ready',
+        'forgot_password submit',
+        'signin submit',
+        'signup submit',
+        'federated login'
+      ];
+      validEvents.forEach(function (e) {
+        instance.on(e, function () {
+          var args = arguments;
+          if (arguments.length === 1) {
+            args = arguments[0];
+          }
+          logs.push({ event: e, arguments: args });
+          printLogs();
+        });
+      });
+    }
+    $('#btn-clear-log').on('click', clearLogs);
+    $(function () {
+      printLogs();
+    });
+  </script>
   <script>
-    const cid = "BWDP9XS89CJq1w6Nzq7iFOHsTh6ChS2b";
+    const clientId = "BWDP9XS89CJq1w6Nzq7iFOHsTh6ChS2b";
     const domain = "brucke.auth0.com";
-    const options = {
+    const defaultOptions = {
       allowShowPassword: true,
       usernameStyle: 'email',
       defaultDatabaseConnection: 'acme',
       prefill: {
         email: 'johnfoo@gmail.com'
-      }
+      },
+      passwordlessMethod: 'code',
+      allowedConnections: ['sms', 'acme']
```

Comment:
Is this just a sample?? In that case I'd use `email` as it's easier to test. SMS requires setup of the phone provider, verification of dev phones, etc. Even more if you think that's our tenant up there ☝️ 😅

Reply:
agree. will change that to email